### DISCUSSION TITLE
제목 1~3의 색상 변경을 하고 싶습니다. 또한 우측 상단의 별 버튼 구독이 안됩니다.

### DISCUSSION THREAD

**Discussion by Dreaming-J** [pronist/hELLO#181] [Q&A] [answered]
**1. 제목 1~3의 글색상이나 배경색을 적용 안됨**
css 코드를 보니 아래 부분으로 인해 색상이 고정된다고 판단했습니다.
하지만 어떻게 수정해야 하는지는 모르겠어서 질문 드립니다.
색상을 자유롭게 설정하고 싶은 경우 어떻게 해야 하는지 알려주실 수 있나요?
<img width="300" height="220" src="https://user-images.githubusercontent.com/78305338/215766481-e74bf595-8059-454e-b317-e294bb978230.png">

**2. 우측 상단에 별 버튼 시 구독 안됨**
 hELLO 스킨 적용 중인 다른 티스토리를 구경하다 별 버튼을 누르니 구독이 되는 것을 알았습니다.
하지만 정상우님의 블로그에 별 버튼은 아무리 눌러도 구독이 되지 않아서 질문 드립니다.
별 버튼으로 구독을 하려면 어떻게 해야 하나요?
그리고 별 버튼 외에는 구독할 수 있는 방법이 없나요?

**Comment by pronist** [comment, marked as answer]
@Dreaming-J 안녕하세요.

**제목 1~3의 글색상이나 배경색을 적용 안됨**

색상은 고정되어 있습니다. `style.css` 를 변경하는 방법 이외에도 티도리 프레임워크를 사용하여 직접 원본 소스코드를 컴파일하여 수정하는 방법도 있습니다. **Tailwind CSS** 가 사용되었으며 원본 소스 코드에서 다음의 위치에 존재합니다.  `style.css` 에서는 제목의 `a` 에 해당하는 태그에 색상을 바꾸십시오.

https://github.com/tistory-projects/tistory-skin-hELLO/blob/47ba71025164adad5d1c9f01f3a63517266417ec/views/Main/Post/Permalink/Post/Content.pug#L231-L239

**우측 상단에 별 버튼 시 구독 안됨**

@Dreaming-J 님의 블로그를 테스트로 구독해본 결과 정상적으로 처리됨이 확인되었습니다.  하지만,  **2차 도메인**에서는 구독 버튼을 비롯한 로그인을 대상으로 하는 활동이 전혀 동작하지 않습니다. <https://pronist.dev> 는 2차 도메인이고, <https://pronist.tistory.com> 이 티스토리 원 도메인입니다. 이에 대한 공지는 티스토리 공식 블로그의 [2차 도메인](https://notice.tistory.com/2546) 관련 공지를 참고해주세요.
